core/domain/draft_upgrade_services.py

```diff
@@ -88,6 +88,36 @@ def try_upgrading_draft_to_exp_version(
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v30_dict_to_v31_dict(cls, draft_change_list):
+        """Converts draft change list from state version 30 to 31. State
+        version 31 adds a customization arg for the "Add" button text
+        in SetInput interaction.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        for i, change in enumerate(draft_change_list):
+            if (change.cmd == exp_domain.CMD_ADD_CUSTOMIZATION_ARG and
+                    change.property_name ==
+                    exp_domain.STATE_PROPERTY_INTERACTION_CUST_ARGS and
```

Comment:
Actually, on further investigation, I have a question about this. Given that SetInput has no customization args at the moment, how can a draft which updates the customization args even arise?

Reply:
Ah, I see. I was under the impression that draft changes include additions along with modifications. I've removed this. Thanks!

Reply:
Ah, I see. I was under the impression that draft changes included additions along with modifications. I've removed this now. Thanks!